### PR TITLE
Fix responsive admin headers

### PR DESCRIPTION
**Pull request by dimitrinicolas**
My PR is a:

💅 Enhancement

Main update on the:
Admin

Fixed a responsive issue on admin title headers `.containerFluid > div:first-child`.

Before:

capture d ecran 2018-11-04 a 12 41 16

After:

capture d ecran 2018-11-04 a 12 41 27

### CODE REVIEW

**Comment by lauriejim** [comment]
Thank you for this PR.
Can you give me the browser you use to have this issue please.

**Comment by dimitrinicolas** [comment]
I use Chrome 70.0.3538.77.`

File: packages/strapi-admin/admin/src/containers/InstallPluginPage/styles.scss [hunk 1]
```diff
@@ -1,8 +1,7 @@
 .containerFluid {
   padding: 18px 30px !important;
   > div:first-child {
-    max-height: 33px;
-    margin-bottom: 48px;
+    margin-bottom: 12px;
```
Comment (soupette) [inline packages/strapi-admin/admin/src/containers/InstallPluginPage/styles.scss]: To keep the alignement `11px` is better
screenshot 2018-11-13 at 09 57 25

Comment (dimitrinicolas) [reply packages/strapi-admin/admin/src/containers/InstallPluginPage/styles.scss]: I will change it in the day 👍

**Comment by dimitrinicolas** [comment]
Just changed value 👍

**Comment by derrickmehaffy** [comment]
Tad late for 14.5 :( but once soupette gives the okay it'll be in the next release.

**Pull request merged**